packages/babel-types/src/definitions/core.js

```diff
@@ -279,41 +302,44 @@ defineType("FunctionExpression", {
   inherits: "FunctionDeclaration",
   aliases: ["Scopable", "Function", "BlockParent", "FunctionParent", "Expression", "Pureish"],
   fields: {
```

Comment:
A FunctionExpression can't have `declare`. `id` and `body` are the same but I don't think it would be worth extracting those out separately.